test/integration/controller/jobset_controller_test.go

```diff
@@ -134,16 +134,16 @@ var _ = ginkgo.Describe("JobSet validation", func() {
 			updates: []*jobSetUpdate{
 				{
 					fn: func(js *jobset.JobSet) {
-						// Try mutating jobs list.
-						js.Spec.ReplicatedJobs = append(js.Spec.ReplicatedJobs, testing.MakeReplicatedJob("test-job").
-							Job(testing.MakeJobTemplate("test-job", ns.Name).PodSpec(testing.TestPodSpec).Obj()).
+						ginkgo.By("mutating jobs list")
+						js.Spec.ReplicatedJobs = append(js.Spec.ReplicatedJobs, testing.MakeReplicatedJob(ns.Name).
```

Comment:
The parameter to MakeReplicatedJob is the name of the ReplicatedJobSet, not the namespace

Reply:
Yea, sorry.  I notice sometimes we get test-job exists so I was trying something.  I'll revert.